troposphere/autoscaling.py

```diff
@@ -149,6 +149,7 @@ class LaunchTemplateOverrides(AWSProperty):
     props = {
         "InstanceType": (str, False),
         "WeightedCapacity": (str, False),
+        "LaunchTemplateSpecification": (LaunchTemplateSpecification, True),
```

Comment:
Thank you for the commit. Could you alphabetize this by moving it up a line? And I believe it should be "not required" so it should be False instead of True.

Reply:
Oops, sorry about that! I've fixed it up now.